app/src/main/java/fr/free/nrw/commons/mwapi/CustomMwApi.java

```diff
@@ -69,6 +69,10 @@ public String getAuthCookie() {
     }
 
     public void setAuthCookie(String authCookie) {
+        if (authCookie == null) {//If the authCookie is null, no need to proceed
```

Comment:
Will the flow be affected as authCookie is not set? Should the user be logged out?

Reply:
I think it should not, isLogged in is set true after that